### ISSUE TITLE
Feature Request: Warps add individual players

### ISSUE THREAD

**Issue by Nutmegan** [CanaryModTeam/CanaryMod#220]
I would like a way to add a list of players that are allowed to use a warp.

**Comment by 14mRh4X0r** [comment]
Couldn't you just put them in a group specifically for that warp?

**Comment by Nutmegan** [comment]
Yes, I currently have to do that for warps and for other special cases but my groups file is becoming a wreck to manage. There is already an owner entry that could simply be made into a list rather than a single entry field.

**Comment by 14mRh4X0r** [comment]
In my personal opinion, adding this to canary plainly duplicates functionality. The fact that your groups file is a wreck might be a good indication you need to start using a database.

Just my 2 cents

**Comment by Nutmegan** [comment]
Why would needing to have two people listed on a warp constitute requiring a group and thus be duplication? I don't want to keep having to create groups just so players can add a second person to their private warps. If owner were a list or there were an option for allowed users, the player could simply add the second owner or user themselves. Using a database isn't going to change that problem or make it go away.
